src/NuGetForUnity/Editor/NugetWindow.cs

```diff
@@ -683,39 +678,29 @@ private void DrawInstalledHeader()
                         }
                     }
 
-                    if (NugetHelper.SelectedPackagesCount > 0)
+                    if (NugetHelper.InstalledPackages.Where(p => p.IsSelected).Count() > 0)
```

Comment:
I think Count can accept a predicate as well so this could be `InstalledPackages.Count(p => p.IsSelected)`

Reply:
```suggestion
                    if (NugetHelper.InstalledPackages.Any(package => package.IsSelected))
```
actually you don't even need to count the packages.